src/v2/view-builder/views/phone/ChallengeAuthenticatorDataPhoneView.js

```diff
@@ -0,0 +1,97 @@
+import {
+  loc,
+  createButton } from 'okta';
+import BaseView from '../../internals/BaseView';
+import BaseForm from '../../internals/BaseForm';
+import BaseFooter from '../../internals/BaseFooter';
+import BaseFactorView from '../shared/BaseFactorView';
+import { addSwitchAuthenticatorLink } from '../../utils/AuthenticatorUtil';
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    className: 'phone-authenticator-challenge',
+    events: {
+      'click a.phone-authenticator-challenge__link' : 'handleSecondaryLinkClick'
+    },
+
+    title () {
+      return loc('oie.phone.verify.title', 'login');
+    },
+
+    save () {
+      return (this.model.get('primaryMode') === 'sms')
+        ? loc('oie.phone.sms.primaryButton', 'login')
+        : loc('oie.phone.call.primaryButton', 'login');
+    },
+
+    saveForm () {
```

Comment:
remove this

Reply:
This is needed to support line #36 (https://github.com/okta/okta-signin-widget/pull/1250/files?file-filters%5B%5D=.js#diff-29ad4a27cc5a8a295fd80957573aa2f1R36)

Reply:
the method body is just calling super (but passing this.model) which doesn't quite necessary.
you shall be able to call  `this.saveForm(this.model)` in that line